Core.DomainModel/Organization/OrganizationUnit.cs

```diff
@@ -150,6 +150,26 @@ public void ConvertToKitosUnit()
             ExternalOriginUuid = null;
         }
 
+        private bool CanBeModified()
```

Comment:
think we need to be more clear here.... the real question is... "CanBeRelocated" ... right?... that's why it checks if it is the root?

Maybe it should be the org answering these questions because that's the one that knows if it is a root or not